ProiectPractica/Controller/ConfigForm.cs

```diff
@@ -0,0 +1,74 @@
+using ProiectPractica.Model;
+
+namespace ProiectPractica
+{
+    public partial class ConfigForm : Form
+    {
+        public ConfigForm()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Still in progress
+        /// </summary>
+        public void SetRepository()
+        {
+            // aici ar trebui sa initializez repo-ul si sa i-au obiectul Configurare
+            // pentru testare am folosit am initializat un obiect Configurare
+            var testConfig = new Configurare
+            {
+                MinAcceptablePrice = 60,
+                MinPricePerKm = 1.7F,
+                NumberOfCars = 2,
+                ReservationCheckInterval = 2,
+                PhoneNumber = "+40747633357",
+                MinPriceForShortTrips = 50,
+                ShortTripDistanceThreshold = 15,
+                StartBusinessHour = 9,
+                EndBusinessHour = 19
+            };
+
+            SetValuesInTextBoxes(testConfig);
+            MakeButtonsUnavailable();
+        }
+
+        /// <summary>
+        /// Auxiliary function that prevents duplicate code.
+        /// Instead of putting values in text boxes like this: textBoxName.Text = value; nine times,
```

Comment:
Este o metoda privata, nu era necesar sa scrii summary. Chiar si asa, nu ar trebui sa specfici detalii de implementare, doar ce face functional metoda ta. Daca cineva chiar vrea sa afle mai multe, poate sa se uite in corpul metodei. Si daca crezi ca codul nu se poate citi usor, poti lasa comentarii succinte.